salt/suricata/soc_suricata.yaml

```diff
@@ -1,6 +1,6 @@
 suricata:
   enabled: 
-    description: You can enable or disable Suricata.
+    description: Enables or disables the Suricata process. This process is used for triggering alerts and optionally for packet meta-data collection and network packet recording.
```

Comment:
Enables or disables the Suricata process. This process is used for triggering alerts and optionally for protocol metadata collection and full packet capture.